packages/react-native/Libraries/Pressability/Pressability.js

```diff
@@ -567,6 +584,7 @@ export default class Pressability {
       onClick: (event: any): void => {
         // If event has `pointerType`, it was emitted from a PointerEvent and
         // we should ignore it to avoid triggering `onPress` twice.
+        debugger;
```

Comment:
👋 this should probably be removed
cc: @yusufyildirim @douglowder

Reply:
Haha, amazing. Thanks for catching that 😄

Reply:
Line has been removed in https://github.com/react-native-tvos/react-native-tvos/commit/4ae06004665d7f72147422b42d2bd65c1e192969